packages/next/README.md

```diff
@@ -1127,6 +1127,51 @@ The `ctx` object is equivalent to the one received in all [`getInitialProps`](#f
 
 - `renderPage` (`Function`) a callback that executes the actual React rendering logic (synchronously). It's useful to decorate this function in order to support server-rendering wrappers like Aphrodite's [`renderStatic`](https://github.com/Khan/aphrodite#server-side-rendering)
 
+#### Customizing `renderPage`
+- It takes as argument an options object for further customization
```

Comment:
These docs should mentioned that the only use-case this should be used for is implementing css-in-js server rendering.